vision/diffusion_mnist/Project.toml

```diff
@@ -0,0 +1,13 @@
+[deps]
+BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+DrWatson = "634d3b9d-ee7a-5ddf-bec9-22491ea816e1"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
+Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+MLDatasets = "eb30cadb-4394-5ae3-aed4-317e484a6458"
+Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
+ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+TensorBoardLogger = "899adc3e-224a-11e9-021f-63837185c80f"
```

Comment:
I used the `vae_mnist.jl` as a template so I copied these over from its `Project.toml` file:
```
[compat]
CUDA = "2.4"
Flux = "0.11.4, 0.12"
julia = "1.5"
```
Not sure if this is what you were looking for 🤷‍♂️

Reply:
I would use the latest version of all packages (you can find this via `] st`). You can use the major version for any post-1.0 libraries and `0.X` for others (including Flux itself). Julia should be at least 1.6 since that's the current LTS.